.changeset/dull-lions-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+"nextjs-website": patch
```

Comment:
```suggestion
"nextjs-website": minor
```
I think is more accurate to set version as `minor` update, because it add a new feature